qiskit/ignis/verification/randomized_benchmarking/rb_utils.py

```diff
@@ -59,34 +64,110 @@ def count_gates(qobj, basis, qubits):
     return ngates
 
 
-def gates_per_clifford(qobj_list, clifford_length, basis, qubits):
-    """Take a list of compiled qobjs (for each seed) and use these
-    to calculate the number of gates per clifford.
+def gates_per_clifford(transpiled_circuits_list: List[List[QuantumCircuit]],
```

Comment:
This isn't really backwards compatible. To support the old interface you would have to do type checking and raise a warning if it is the old type. I think you should do:

```python
def gates_per_clifford(transpiled_circuits_list: Union[List[List[QuantumCircuit]], List[QasmQobj]],  # Qobj Deprecated
                       clifford_lengths: Union[np.ndarray, List[int]],     # Np.array deprecated
                       basis: List[str],
                       qubits: List[int]
                       ) -> Dict[int, Dict[str, float]]:
if isinstance(tanspiled_circuits_list[0], QasmQobj):
    # raise deprecation warning stating that qobj input is deprecated, use transpiled circuit objects instead.
    ...
```

Then it has the same number of arguments as the original function, so is backwards compatible with legacy code, but will raise a deprecation warning if you are using the old types.

Reply:
ok, they are just for user using **kwargs. I'll remove.